### PR TITLE
[JENKINS-34139] - Document potential requirements to masters in CpsScmFlowDefinition

### PR DESCRIPTION
**Pull request by oleg-nenashev**
https://issues.jenkins-ci.org/browse/JENKINS-34139

@reviewbybees

### CODE REVIEW

File: src/main/resources/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition/help-scm.html [hunk 1]
```diff
@@ -4,4 +4,12 @@
     (If you wish to use other files from the same repository during your Pipeline,
     you will need to check them out separately on some slave;
     this checkout cannot be reused.)
+
```
Comment (jglick) [inline src/main/resources/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition/help-scm.html]: :ant: I think you forgot a `<p>` here.

**Comment by jglick** [comment]
Rather than guiding the user toward satisfying these snowflake conditions on the Jenkins master, it is better advice to remind them that `CpsScmFlowDefinition` as a whole is merely a convenience, and that [you can customize the behavior, including selecting a specific node, by doing the checkout explicitly](https://github.com/jenkinsci/pipeline-plugin/blob/72a4ac9c280013c8a68a7b8b24b9b1a8d6ea045a/TUTORIAL.md#triggering-manual-loading).

**Comment by amuniz** [comment]
:bee:

**Comment by jglick** [comment]
:bug: as above

**Comment by oleg-nenashev** [comment]
@jglick I barely remember what it was about. Do you think https://issues.jenkins-ci.org/browse/JENKINS-34139 worth fixing in this or any other way?

**Pull request closed without merging**